addons/web/static/src/views/fields/selection/selection_field.js

```diff
@@ -13,6 +13,7 @@ export class SelectionField extends Component {
         placeholder: { type: String, optional: true },
         required: { type: Boolean, optional: true },
         domain: { type: Array, optional: true },
+        autosave: { type: Boolean, optional: true },
```

Comment:
You should add tests in web/ for this props and a test with selection field in kanban